analyzers/src/SonarAnalyzer.CSharp/Rules/CryptographicKeyShouldNotBeTooShort.cs

```diff
@@ -220,7 +221,7 @@ private static void CheckBouncyCastleParametersGenerators(ITypeSymbol containing
 
```

Comment:
Line 104 and 105 can be joined https://github.com/SonarSource/sonar-dotnet/blob/e84d7a22fc362fa4a582c9869fb81cb81c34ab2c/analyzers/src/SonarAnalyzer.CSharp/Rules/CryptographicKeyShouldNotBeTooShort.cs#L103-L106

```cs
 if (GetPropertyName(assignment.Left) == "KeySize" 
     && assignment.Left is MemberAccessExpressionSyntax { Expression: { }  expression }
     && c.SemanticModel.GetTypeInfo(expression).Type is ITypeSymbol containingType) 
```

Reply:
There are some property names hardcoded in the file like `"KeySize"`. Please check if `nameof` can be used instead.